stl/inc/optional

```diff
@@ -53,10 +53,14 @@ protected:
     _THROW(bad_optional_access{});
 }
 
+struct _Nontrivial_Dummy_type {
+    constexpr _Nontrivial_Dummy_type() noexcept {}
+};

```

Comment:
Would it be a good idea to add an STL-internal static_assert that this is nontrivial? Ok if the answer is meh.

Reply:
Meh, but why not.